tests/src/test/java/io/streamnative/pulsar/handlers/kop/SchemaRegistryTest.java

```diff
@@ -69,7 +60,7 @@ protected void setup() throws Exception {
         bootstrapServers = "localhost:" + getKafkaBrokerPort();
     }
 
-    @BeforeMethod
+    @BeforeMethod(alwaysRun = true)
```

Comment:
It's a typo written by me for the initial commit. We should use `@AfterMethod`.

Reply:
good catch